### ISSUE TITLE
Footer: add copy about Creative Commons

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4755]
Abigail from legal has asked us to add a sentence to all our footers about Creative Commons. She gave this example https://wikimediafoundation.org/ but mozilla.org also has some copy.

### Todo
- [ ] Get final copy from Abigail
- [ ] Mock-up the footer with the new copy for mobile, tablet, desktop
- [ ] List all affected sites (foundation, both donate sites, pulse, mozfest, pni, anything else?)
- [ ] Get approval from Abigail and give her a timeline for implementation
- [ ] Get R+ from a designer
- [ ] Check-in with devs 
- [ ] Merge updates to Abstract
- [ ] Open implementation tickets

cc @beccaklam @taisdesouzalessa @mmmavis

**Comment by mmmavis** [comment]
Is this something we wanna do to all other MoFo sites as well? (Pulse & Donate)

**Comment by kristinashu** [comment]
@mmmavis I listed foundation, both donate sites, pulse, and mozfest. Can you think of any others or any reason to not update the ones listed? We could also check with Abigail once we have the comprehensive list.

**Comment by mmmavis** [comment]
@kristinashu no concern from me. Mainly just wanted to clarify.

**Comment by taisdesouzalessa** [comment]
@mmmavis tickets for alignment of donate footer with other existing footers are [here](https://github.com/mozilla/donate-wagtail/issues/1018) and [here](https://github.com/mozilla/donate-wagtail/issues/1017). 

Once we have the copy from Abigail on Creative Commons, I'll work on the mockups with the new copy included.

**Comment by taisdesouzalessa** [comment]
Sent an email to Abigail requesting the copy + provided the list of all web properties affected so she is aware of all the impacted sites.
Waiting on her feedback to proceed with this work.

**Comment by taisdesouzalessa** [comment]
Here are the options with placeholder copy while we wait for Abigail's answer: https://mozilla.invisionapp.com/share/PJXWEC7BU86 - submitted to designers for feedback.

**Comment by taisdesouzalessa** [comment]
Abigail will have some copy ready by EOD today/before Monday morning.

**Comment by taisdesouzalessa** [comment]
During this sprint we advanced conversations on this issue, figure out placement for copy and we are now very close to the final copy.

After talking to Abigail via email, she recommended we get a new page up with a fuller explanation of the license. Since this task got a new requirement, I created a new ticket to be tackled next sprint: #4868

Closing this ticket
